main.py

```diff
@@ -1,10 +1,70 @@
 import init_django_orm  # noqa: F401
+import json
+
+from typing import Dict
 
 from db.models import Race, Skill, Player, Guild
 
 
+def reader(filename: str) -> Dict:
+    with open(filename) as file:
+        return json.load(file)
+
+
+def create_or_update_race(players_race: dict) -> Race:
+    race, _ = Race.objects.get_or_create(
+        name=players_race["name"],
+        defaults={"description": players_race["description"]}
+    )
+    return race
+
+
+def create_or_update_skills(players_race: dict, race_instance: Race) -> None:
+    for skill in players_race:
+        skill_instance, _ = Skill.objects.get_or_create(
+            name=skill["name"],
+            bonus=skill["bonus"],
+            race=race_instance
+        )
+
+
+def create_or_update_guild(player_guild: dict) -> Guild | None:
+    if player_guild:
+        guild, _ = Guild.objects.get_or_create(
+            name=player_guild["name"],
+            defaults={"description": player_guild["description"]}
+        )
+        return guild
+    return None
```

Comment:
no need to return None explicitly - if you dont specify return, function will return None by default